libs/ast_generic/Id_info_flags.mli

```diff
@@ -0,0 +1,29 @@
+(* Andre Kuhlenschmidt
+ *
+ * Copyright (C) 2023 Semgrep Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file LICENSE.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * LICENSE for more details.
+ *)
+
+type packed
+type unpacked = { hidden : bool; case_insensitive : bool }
```

Comment:
maybe better to move the documentation about hidden from the .ml to the .mli (it used to be in ast_generic.ml,
so in a .ml, but there's no ast_generic.mli because the file was mostly type definitions that is annoying to duplicate). You can also add a quick comment about case_insensitive here in the .mli such as
(* this is useful for languages such as PHP or Apex which use case_insensitive names *)